lib/shared/sdk/index.js

```diff
@@ -192,28 +192,28 @@ SDK.Scanner = class Scanner extends EventEmitter {
   }
 
   /**
-   * @summary Subsribe to an adapter
+   * @summary subscribe to an adapter
```

Comment:
Minor nit: should there be a capital 'S' on 'subscribe' here?

Reply:
Yup, fixed! The pleasures of global find & replace tricked me here.

Reply:
Sometimes those find'n'replace dialogs have a handy "match exact case only" tickbox :stuck_out_tongue_winking_eye: